cegui/include/CEGUI/RendererModules/Ogre/Renderer.h

```diff
@@ -111,6 +111,9 @@ struct OgreRenderer_impl;
 class OGRE_GUIRENDERER_API OgreRenderer : public Renderer
 {
 public:
+    //!	Convenience function that configures a CEGUI::Window to display an Ogre RenderToTexture (RTT) 
+    void OgreRenderer::configureCeguiWindowForRTT(CEGUI::Window* window, const std::string& ogreTextureName, float textureWidth, float textureHeight);
```

Comment:
`configureCeguiWindowForRTT` not `OgreRenderer::configureCeguiWindowForRTT`

Reply:
corrected